Application/app/src/template/res/values-de/strings.xml

```diff
@@ -14,7 +14,7 @@ express or implied. See the License for the specific language governing
 permissions and limitations under the License.
 -->
 <resources>
-  <string name="app_name"><APP_NAME></string>
+  <string name="app_name">Test app</string>
```

Comment:
We should not replace placeholders for 'template' build variant here and in other 'template' files